client/components/OrderItem.js

```diff
@@ -1,22 +1,28 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
-import { deleteOrderItemThunk } from '../store';
+import { deleteOrderItemThunk, fetchProductVariants } from '../store';
 
 class OrderItem extends Component {
   constructor() {
     super();
     this.state = {};
   }
 
+  componentDidMount() {
+    this.props.fetchProductVariants();
+  }
+
   render() {
-    const { orderItem, product, userId, orderId } = this.props;
+    const { orderItem, product, userId, orderId, productVariants } = this.props;
     const { price, quantity } = orderItem;
+
     if (!orderItem.price) {
-      orderItem.price = product.price;
+      orderItem.price = product.productvariant[0].price;
```

Comment:
Is this setting the order item price to the first product variant if it doesn't have one? Is that something we want to do?